pretix_eth/templates/pretix_eth/checkout_payment_form.html

```diff
@@ -1,16 +1,30 @@
 {% load bootstrap3 %}
 <div class="form-horizontal">
-    {% if ETH_address %}
-      <div class="ETH-crypto-payment-details">
-        <p>Send {{ ETH_per_ticket }} ETH to {{ ETH_address }}</p>
-      </div>
-    {% endif %}
-    {% if DAI_address %}
-      <div class="ETH-crypto-payment-details">
-        <p>Send {{ DAI_per_ticket }} DAI to {{ DAI_address }}</p>
-      </div>
-    {% endif %}
-
-
-    {% bootstrap_form form layout='horizontal' %}
+  {% if ETH_address %}
+  <div class="ETH-crypto-payment-details">
+    <p>Send {{ ETH_per_ticket }} ETH to {{ ETH_address }}</p>
+    <a
+      href="https://checkout.web3connect.com/?currency=ETH&amount={{
```

Comment:
does this need to be a 3rd party site or can it be a static site living here?

Reply:
I don't see any problem being an external website just like a Paypal payment would happen. But feel free to fork and include it in the plugin if you have time.

Source code: https://github.com/pedrouid/web3connect-checkout

Reply:
It adds a point of failure. In the case of paypal it is needed - I do not see the need here.

Reply:
The problem was (as @pipermerriam described on Discord) the content-security policy of templates rendered by Django broke the JS logic which broke the libraries we used like Web3Connect

I will leave it up to you guys to include this PR in the first wave or not.

Reply:
yea - this is for templates - but does this also apply for static sites? cc @pipermerriam

Reply:
I believe it's going to be pages served by django.

Reply:
@ligi Django should be able to serve a static site.